Tests/Core/JWTTests.swift

```diff
@@ -53,7 +53,7 @@ class JWTTests: QuickSpec {
                 expect { try proxy.toJSON(string: "blah.bl{ah.blah") }.to(throwError(JWTHelperError.componentDecodingError("")))
             }
 
-            fit("Should fail on invalid json in second jwt component") {
```

Comment:
what is fit?

Reply:
It makes "only this test" run :p somehow got in from a previous commit.